docs/developer-docs/web-apps/user-login/internet-identity/integrate-internet-identity.mdx

```diff
@@ -112,17 +130,24 @@ Open the `dfx.json` file and replace the existing content with the following:
 }
 ```
 
-</TabItem>
-</Tabs>
+Pull the II canister using `dfx deps`:
 
+```
+dfx deps pull
+```
 
-### Step 2: Start `dfx`:
+Initialize the canister. You can use the `'(null)'` value passed to the init command to use the default values. To do so, run the command:
 
 ```
-dfx start --clean --background
+dfx deps init internet_identity --argument '(null)'
```

Comment:
`dfx 0.18.0` actually automatically intializes the argument to `(null)`, so no need to specify this explicitly. Although `dfx deps init` still has to be invoked, see [here](https://dfinity.slack.com/archives/CGA566TPV/p1710148272653119).

Reply:
The instructions in this are for dfx 0.16.0 and older, since in 0.17.0 and newer, II can be added to the project as part of the `dfx new` command instead.